crates/devices/virtio-blk/Cargo.toml

```diff
@@ -13,12 +13,12 @@ edition = "2018"
 backend-stdio = []
 
 [dependencies]
-vm-memory = ">=0.8.0"
-vmm-sys-util = ">=0.8.0"
-log = ">=0.4.6"
+vm-memory = "0.9.0"
+vmm-sys-util = "0.10.0"
+log = "0.4.17"
 virtio-queue = { path = "../../virtio-queue" }
 virtio-device = { path = "../../virtio-device" }
-virtio-bindings = { path = "../../virtio-bindings", version = ">=0.1.0" }
+virtio-bindings = { path = "../../virtio-bindings", version = "0.1.0" }
 
 [dev-dependencies]
 vm-memory = { version = ">=0.8.0", features = ["backend-mmap", "backend-atomic"] }
```

Comment:
Was it intentional to leave this as ">=0.8.0" and not "0.9.0" ?

Reply:
I think this was a mistake. We should fix it, good catch!